tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml

```diff
@@ -45,15 +45,16 @@
                         IsPaneToggleButtonVisible="False"
                         IsTitleBarAutoPaddingEnabled="False"
                         OpenPaneLength="218"
+                        MaxHeight="{StaticResource ContentDialogMaxHeight}"

```

Comment:
The right value should be `654`

Reply:
Sorry I misunderstood the height☹️, thanks for correcting. But maybe 652 is better (?)

Reply:
Could works too.